assets/app/scripts/extensions/javaLink.js

```diff
@@ -40,25 +40,27 @@ angular.module('openshiftConsoleExtensions', ['openshiftConsole'])
       if (!jolokiaPort) {
         return;
       }
-      var pod = $scope.$eval('pod');
+      var pod = $scope.$eval('podTemplate');
```

Comment:
Hmm, best way to distinguish then?  As of a previous PR the java console link disappeared, this change brought it back...